src/main/java/duke/UI.java

```diff
@@ -0,0 +1,49 @@
+package duke;
+
+import java.util.Scanner;
+
+/**
+ * The UI class to handle user interactions
+ */
+public class UI {
```

Comment:
Abbreviations and acronyms should not be uppercase when used as a part of a name. We can change the name to Ui.